src/lesson_4/FractionalWrapper.sol

```diff
@@ -152,7 +152,15 @@ contract FractionalWrapper is ERC20 {
     ///@param owner the owning address
     ///@return shares amount of vault asset
     function withdraw(uint256 assets, address receiver, address owner) public returns (uint256 shares) {
-        shares = _convertToShares(assets);
+        shares = previewWithdraw(assets);
```

Comment:
`previewWithdraw` just calls `_convertToShares` which is internal. You don't need to change this line.

Reply:
Fixed.